.github/workflows/beam_LoadTests_Go_GBK_Flink_Batch.yml

```diff
@@ -0,0 +1,166 @@
+          CURCONFIG=$(echo "${CURCONFIG/load-tests-go-flink-batch-gbk-6-/load-tests-go-flink-batch-gbk-6-$CURDATE}")
+          echo "prepared_config_6=$CURCONFIG" >> $GITHUB_OUTPUT
+      - name: Start Flink with parallelism 5
+        env:
+          FLINK_NUM_WORKERS: 5
+        run: |
+          cd ./.test-infra/dataproc; ./flink_cluster.sh create
+      - name: run GBK Flink Batch Go Load Test 1 (10 b records)
+        uses: ./.github/actions/gradle-command-self-hosted-action
+        with:
+          gradle-command: :sdks:go:test:load:run
+          arguments: |
+            -PloadTest.mainClass=group_by_key \
+            -Prunner=FlinkRunner \
+            '-PloadTest.args=${{ steps.set_configs.outputs.prepared_config_1 }}' \
+      - name: run GBK Flink Batch Go Load Test 2 (100 b records)
+        uses: ./.github/actions/gradle-command-self-hosted-action
+        with:
+          gradle-command: :sdks:go:test:load:run
+          arguments: |
+            -PloadTest.mainClass=group_by_key \
+            -Prunner=FlinkRunner \
+            '-PloadTest.args=${{ steps.set_configs.outputs.prepared_config_2 }}' \
+      - name: run GBK Flink Batch Go Load Test 3 (100 kb records)
+        uses: ./.github/actions/gradle-command-self-hosted-action
+        with:
+          gradle-command: :sdks:go:test:load:run
+          arguments: |
+            -PloadTest.mainClass=group_by_key \
+            -Prunner=FlinkRunner \
+            '-PloadTest.args=${{ steps.set_configs.outputs.prepared_config_3 }}' \
+      - name: run GBK Flink Batch Go Load Test 6 (reiterate 4 times 10 kb)
+        uses: ./.github/actions/gradle-command-self-hosted-action
+        with:
+          gradle-command: :sdks:go:test:load:run
+          arguments: |
+            -PloadTest.mainClass=group_by_key \
+            -Prunner=FlinkRunner \
+            '-PloadTest.args=${{ steps.set_configs.outputs.prepared_config_6 }}' \
+      - name: Restart Flink with parallelism 16
+        env:
+          FLINK_NUM_WORKERS: 16
+        run: |
+          cd ./.test-infra/dataproc; ./flink_cluster.sh restart
+      - name: run GBK Flink Batch Go Load Test 4 (fanout 4)
+        uses: ./.github/actions/gradle-command-self-hosted-action
+        with:
+          gradle-command: :sdks:go:test:load:run
+          arguments: |
+            -PloadTest.mainClass=group_by_key \
+            -Prunner=FlinkRunner \
+            '-PloadTest.args=${{ steps.set_configs.outputs.prepared_config_4 }}' \
+      - name: run GBK Flink Batch Go Load Test 5 (fanout 8)
+        uses: ./.github/actions/gradle-command-self-hosted-action
+        with:
+          gradle-command: :sdks:go:test:load:run
+          arguments: |
+            -PloadTest.mainClass=group_by_key \
+            -Prunner=FlinkRunner \
+            '-PloadTest.args=${{ steps.set_configs.outputs.prepared_config_5 }}' \
+      - name: Teardown Flink
+        if: always()
```

Comment:
What happens in this script if this runs but the setup did not run? Does it just fail or are there side effects?

Reply:
It will just fail with cluster NOT_FOUND error